README.md

```diff
@@ -12,7 +12,7 @@ This web application is built to display covid-19 data of countries affected by
 
 ## Live Demo
 
-[View Life](https://duls-travelers-hub.herokuapp.com/)
+[View Life](https://duls-covid-reporter.herokuapp.com/)
```

Comment:
- [ ] let's add a recorded video as requested in the guidelines

Reply:
This issue is well resolved! ✔️